Kitodo/src/main/java/org/kitodo/config/xml/fileformats/FileFormatsConfig.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.config.xml.fileformats;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.nio.file.FileSystems;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+
+import org.apache.myfaces.util.FilenameUtils;
+import org.kitodo.api.filemanagement.FileManagementInterface;
+import org.kitodo.config.Config;
+import org.kitodo.serviceloader.KitodoServiceLoader;
+
+/**
+ * A {@code kitodo_fileFormats.xml} config file. This class corresponds to the
+ * {@code <kitodo_fileFormats>} tag in {@code kitodo_fileFormats.xml}.
+ */
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "", propOrder = {"fileFormat" })
+@XmlRootElement(name = "kitodo_fileFormats")
+public class FileFormatsConfig {
+
+    private static final URI CONFIG_FILE_URI = FileSystems.getDefault()
+            .getPath(FilenameUtils.concat(Config.getKitodoConfigDirectory(), "kitodo_fileFormats.xml")).toAbsolutePath()
+            .toUri();
+
+    @XmlElement(required = true)
+    protected List<FileFormat> fileFormat;
+
+    /**
+     * Returns the list of configured file formats.
+     *
+     * @return the configured file formats
+     * @throws IOException
+     *             if reading from the file fails
+     * @throws JAXBException
+     *             if the content of the file is syntactically or semantically
+     *             incorrect
+     */
+    public static List<FileFormat> getFileFormats() throws IOException, JAXBException {
+        KitodoServiceLoader<FileManagementInterface> fileManagementInterface = new KitodoServiceLoader<>(
```

Comment:
Accessing local application configuration files should not be done over the file management module as this is only for accessing meta data and pictures which could be delivered from a remote repository in an other implementation.